recipes/s6-init/s6-init/init.stage1

```diff
@@ -41,7 +41,19 @@ background
   fdmove -c 2 1
 
   # Exec into stage2 script
-  /etc/rc.init $SERVICE
+  #/etc/rc.init $SERVICE
```

Comment:
Just remove the line. I think we should keep this commit in the PR. System that care about not hanging in the shell should hopefully have a watchdog design that reboots the system.

Reply:
OK, I was a little worried that this would/could be a security issue. But it really is useful for debugging.

Reply:
Hmm... I guess it could become a security issue. If there is possibility for non-root users to write content which triggers this failure.  Probably best to leave it out.

Reply:
It would be nice to have it as a USE flag (which developers can then leave on in local.conf).

Reply:
Yeah, maybe that use flag should just be a string naming something to exec into if /etc/rc.init fails, which can then default to /bin/false. Dunno, I'll try something when I update the PR after Laurent makes some bugfix releases, then we'll see.

Reply:
For now I went with adding exec'ing into /etc/rc.init-fallback, which is essentially just a copy of the system-console service - with a twist, as explained in 0326aed.